website/docs/r/federated_database_instance.html.markdown

```diff
@@ -96,13 +96,13 @@ resource "mongodbatlas_federated_database_instance" "test" {
 
 * `project_id` - (Required) The unique ID for the project to create a Federated Database Instance.
 * `name` - (Required) Name of the Atlas Federated Database Instance.
-  ### `cloud_provider_config` - (Optional) Cloud provider linked to this data federated instance.
```

Comment:
pass by formatting fix in [federated_database_instance](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_database_instance#argument-reference) docs